.shared_rubocop.yml

```diff
@@ -114,6 +120,11 @@ Rails/UnknownEnv:
 Rails/Output:
   Enabled: true
 
+Rails/RootPathnameMethods:
+  Enabled: true
+  Exclude:
+    - 'spec/rails_helper.rb'
```

Comment:
this has some boilerplate that we don't want to change even though it fails rubo